doc/1/getting-started/kotlin/index.md

```diff
@@ -0,0 +1,126 @@
+---
+code: false
+type: page
+title: Kotlin
+description: Kotlin Getting started
+order: 99
+---
+
+# Getting Started
+
+In this tutorial you will learn how to install the Kuzzle **Jvm SDK**.
+This page shows examples of scripts that **store** documents in Kuzzle, and of scripts that subcribe to real-time **notifications** for each new document created.
+
+::: success
+Before proceeding, please make sure your system meets the following requirements:
+
+- **Oracle JDK or OpenJDK** version 8 or higher ([OpenJDK installation instructions](https://openjdk.java.net/install/))
+- A running Kuzzle Server ([Kuzzle installation guide](/core/2/guides/essentials/installing-kuzzle))
+
+:::
+
+
+::: info
+Having trouble? Get in touch with us on [Discord](http://join.discord.kuzzle.io)!
+:::
+
+## Installation
+
+You can find the SDK JARs directly on [bintray](https://bintray.com/kuzzle/maven/kuzzle-sdk-jvm). Download and add it to your classpath.
+
+::: info
+The following examples are made to be executed without any IDE.
+If you're using Eclipse, IntelliJ or another Java IDE, you need to add the SDK as a project dependency in your classpath.
+:::
+
+### Installing on Android using gradle
+
+In your app build.gradle add the following line and sync.
+
+    implementation "io.kuzzle:kuzzle-sdk-jvm:1.0.0"
+
+## First connection
+
+Initialize a new Java project, create a `gettingstartedfirstconnection.kt` file and start by adding the code below:
+
+<<< ./snippets/firstconnection.kt
+
+This program initializes the Kuzzle Server storage by creating a index, and a collection inside it
+Run the program with the following command:
+
+```bash
+$ javac -classpath ./path/to/the/sdk.jar gettingstartedfirstconnection.java
+$ java -classpath .:./path/to/the/sdk.jar gettingstartedfirstconnection
+Connected!
+Index nyc-open-data created!
+Collection yellow-taxi created!
+```
+
+Congratulations, you performed your first connection to Kuzzle Server via a Java program.
+You now know how to:
+
+- Instantiate Kuzzle SDK and connect to Kuzzle Server using a specific protocol (here `websocket`)
+- Create a index
+- Create a collection within an existing index
+
+## Create your first document
+
+Now that you successfully connected to your Kuzzle Server instance, and created an index and a collection, it's time to manipulate some data.
+
+Here is how Kuzzle structures its storage space:
+
+- indexes contain collections
+- collections contain documents
+  Create a `gettingstartedstorage.kt` file in the playground and add this code:
+
+<<< ./snippets/document.kt
+
+As you did before, build and run your program:
+
+```bash
+$ javac -classpath ./path/to/the/sdk.jar  gettingstartedstorage.java
```

Comment:
Files in Kotlin should start with a capital and be CamelCase and shouldn'th this be `gettingstartedstorage.kt`?